AppCenterAnalytics/AppCenterAnalytics/MSAnalytics.m

```diff
@@ -351,6 +359,26 @@ - (MSAnalyticsTransmissionTarget *)transmissionTargetForToken:(NSString *)transm
   return transmissionTarget;
 }
 
+- (void)pauseTransmissionTargetForToken:(NSString *)token {
+  if (self.oneCollectorChannelUnit) {
```

Comment:
if null means disabled? In Android we emit an error log when calling APIs while service is disabled

Reply:
This is an implementation difference. Unfortunately Apple SDK doesn't release any channel resources on disabled state. This will only be called if targets are enabled.

Reply:
Error logs were added.